auth-web/src/services/payment.services.ts

```diff
@@ -301,6 +310,24 @@ export default class PaymentService {
     return axios.get(url, { params })
   }
 
+  static async getEFTShortnameHistoryAndRefunds (shortNameId: string, filterParams: EFTTransactionFilterParams) {
+    const eftShortnameHistory = await PaymentService.getEFTShortnameHistory(shortNameId, filterParams)
```

Comment:
Let's not combine the history call at all, we should update the refund routes with any thing you need.

Reply:
I think since you're just calling EFT refunds once now, makes sense -^